frontend/src/components/Header.tsx

```diff
@@ -114,30 +114,24 @@ const HeaderNoCtx: React.FC<ContextType> = (props) => {
     path: '#',
     exact: false,
     nested: [
-      {
-        title: 'Manage Users',
-        path: '/users',
-        users: GLOBAL_ADMIN,
-        exact: true
-      },
       {
         title: 'Manage Organizations',
```

Comment:
I think that "My Organizations" won't be shown to a global admin, since they don't apply to `ORG_USER | ORG_ADMIN` ((ORG_USER | ORG_ADMIN) & GLOBAL_ADMIN = 0)